src/cli.js

```diff
@@ -250,7 +250,8 @@ async function runCmd (argv, stdout, stderr) {
           transpileOnly: args["--transpile-only"],
           license: args["--license"],
           quiet,
-          target: args["--target"]
+          target: args["--target"],
+          filterAssetBase: parse(process.cwd()).root
```

Comment:
This seems dangerous to change the default behavior because you might include files above the current directory on accident.

Imagine you have `/Users/me/node_modules` and you run this in `/Users/me/mymonorepo/someapp`. You might accidentally include `/Users/me/node_modules`

How about exposing this as a CLI option like `--base`

Reply:
so this is actually how yarn monorepos work. Dependencies in sub packages are hoisted to the root packages node_modules. So it's actually how you want it to work.

Reply:
When using [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/) the `filterAssetBase` should be the root of the monorepo. 

However, the code here is setting `filterAssetBase` to the root of the filesystem, which is incorrect.